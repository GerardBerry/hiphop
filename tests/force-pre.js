"use hopscript"

var hh = require("hiphop");

try {
var prg = <hh.module>
  <hh.outputsignal name="O" init_value=0 />
  <hh.emit signal_name="O" args=${hh.value("O")}/>
</hh.module>;
}catch(e){console.log(e.message)}
