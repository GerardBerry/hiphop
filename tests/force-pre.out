*** SIGNAL ERROR on signal `O`
*** [LOC DISABLED --- run hop -g to get location]
*** Can't get current value of a signal to update itself.
